cfg/builder/builder_finalize.cc

```diff
@@ -304,18 +314,16 @@ void CFGBuilder::computeMinMaxLoops(core::Context ctx, const CFG::ReadsAndWrites
 
         for (const auto &expr : bb->exprs) {
             auto what = expr.bind.variable;
-            const auto minIt = cfg.minLoops.find(what);
-            const auto maxIt = cfg.maxLoopWrite.find(what);
-            int curMin = minIt != cfg.minLoops.end() ? minIt->second : INT_MAX;
-            int curMax = maxIt != cfg.maxLoopWrite.end() ? maxIt->second : 0;
```

Comment:
Note: `maxLoopWrite` is now initialized to `0` when we resize it to size `maxVariableId`.